apps/artemis-web/pages/api/config.ts

```diff
@@ -6,8 +6,10 @@ import {
 } from '../../definitions';
 import auth from '../../middleware/auth';
 import { csrf } from '../../libs/csrf';
+import limiter from '../../middleware/limiter';
 
 const handler = nc()
+  .use(limiter(__filename))
```

Comment:
how will a `__filename`-based rate-limiter behave in practice? It will hit the limit when this particular component is invoked more than `limit` times? Same for the rest filename-based refs (`configs`, `download_tables`, `hijack`, 'rmq_*` etc.)